docs/source/metrics.rst

```diff
@@ -561,6 +561,30 @@ bleu_score [func]
 .. autofunction:: pytorch_lightning.metrics.functional.nlp.bleu_score
     :noindex:
 
+*********************
+Information Retrieval
+*********************
+
+Class Metrics (IR)
```

Comment:
what is the IR for?
```suggestion
Class Metrics
```

Reply:
IR = Information Retrieval. Anyway, it is a typo.

Reply:
Yes, looks good to me !

Reply:
I've called the entry `Information Retrieval Metrics` instead of only `Information Retrieval` as the others in last commit #e95b8d7